tests/functional/send_sms.js

```diff
@@ -145,7 +144,7 @@
         .then(openPage(SEND_SMS_URL, SELECTOR_SEND_SMS_HEADER))
         .then(testElementExists(SELECTOR_MARKETING_LINK))
         .then(click(SELECTOR_LEARN_MORE))
-        .switchToWindow(LEARN_MORE_WINDOW_HANDLE)
+        .then(switchToWindow(1))
```

Comment:
I tried updating the helper function, but that looked like it was going to be a challenge.